.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "rules": {
+    "import/no-default-export": ["warn"],
+    "unicorn/filename-case": ["warn", {
+      "case": "kebabCase"
+    }],
+    "import/first": 0
+  },
+  "plugins": ["unicorn"]
```

Comment:
I didn't realise you need to install a plugin here. Would it be simpler to use. https://www.npmjs.com/package/eslint-plugin-filenames

Reply:
we would need to install the `eslint-plugin-filenames` plugin instead. The advantage of `eslint-plugin-unicorn` for this case is that it actually accepts the option as `kebabCase` for the filename. The `filenames` plugin accepts a regex, which would look like this:
`"filenames/match-regex": ["warn", "^[a-z-]+$", true]`